filters/auth/oidc.go

```diff
@@ -38,8 +40,31 @@ const (
 	stateValidity       = 1 * time.Minute
 	oidcInfoHeader      = "Skipper-Oidc-Info"
 	cookieMaxSize       = 4093 // common cookie size limit http://browsercookielimits.squawky.net/
+
+	// Deprecated: The host of the Azure Active Directory (AAD) graph API
+	azureADGraphHost = "graph.windows.net"
 )
 
+type distributedClaims struct {
+	ClaimNames   map[string]string      `json:"_claim_names"`
+	ClaimSources map[string]claimSource `json:"_claim_sources"`
+}
+
+type claimSource struct {
+	Endpoint    string `json:"endpoint"`
+	AccessToken string `json:"access_token,omitempty"`
+}
+
+// The host of the Microsoft Graph API
+var microsoftGraphHost = "graph.microsoft.com" // global for testing purposes
```

Comment:
not a particular nice solution but @szuecs no good idea how to intercept that API call with the unit test...